pkgs/tools/misc/flameshot/default.nix

```diff
@@ -7,6 +7,8 @@
 , qtsvg
 , nix-update-script
 , fetchpatch
+, kguiaddons
+, withWaylandClipboard ? true
```

Comment:
Is this disabling any features or having any downside on x11 other than having this extra dependency? If not, I think we should just enabled it by default and not bother with an extra option here

Reply:
I _think_ not, as ArchLinux has this enabled by default. What makes me a bit doubtful is that this default isn't upstreamed.

I'd rather keep the option for anyone who runs into an issue. Using overrideAttrs to remove array entries seems fiddly to me. We can always remove the option once upstream removes it (or makes it the default).

I'll comment in the flameshot thread about making this the default.

If keeping the option is a no-go for now I'm also happy to remove it, as I personally have no use for it 😅

Reply:
If we don't know why we need the option, then we can just remove it for now and add it when we need it.